file-formats/sktd/sktd-v1.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/sktd/sktd-v1.json",
+  "title": "Knowledge Transfer Document Properties",
+  "description": "KTD properties",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Format version",
+      "type": "string"
+    },
```

Comment:
Just a reminder: If the `formatVersion` has been introdcued, the schema file needs to be regenerated...